src/TypesAndInterfaces/config-sync/input-config-sync/profile-field-config.ts

```diff
@@ -101,7 +112,7 @@ export const EDIT_PROFILE_FIELDS_ADMIN:InputField[] = [
     ...EDIT_PROFILE_FIELDS,
     new InputSelectionField({title: 'Gender', field: 'gender', type: InputType.SELECT_LIST, required: true, selectOptionList: Object.values(GenderEnum)}),
     new InputField({title: 'Date of Birth', field: 'dateOfBirth', type: InputType.DATE, value: getDateYearsAgo().toISOString(), required: true, validationRegex: DATE_REGEX, validationMessage: 'Required, must be valid age.' }),
-    new InputRangeField({title: 'Walk Level', field: 'walkLevel', required: true, minValue: 0, maxValue: 10, type: InputType.RANGE_SLIDER, validationRegex: new RegExp(/[0-9]+/), validationMessage: 'Required, between 1-10.'}),
+    new InputRangeField({title: 'Walk Level', field: 'walkLevel', required: true, minValue: 1, maxValue: 10, type: InputType.RANGE_SLIDER, validationRegex: new RegExp(/[0-9]+/), validationMessage: 'Required, between 1-10.'}),
```

Comment:
Glad you caught this; let's also adjust redux to ` validationRegex: new RegExp(/[1-9]{1}/)`